docs/tutorial/data-flow.md

```diff
@@ -0,0 +1,153 @@
+## Data Flow
+
+There are several methods for sending data between controllers and components. This tutorial provides an overview of the different possibilities and best practices for your application.
+
+## From parents to children
```

Comment:
Please add some text, e.g. a short sentence explaining which section is relevant for which case

Reply:
Where text? 🦧

Reply:
Please add text below this headline